Sources/Ledger/Classes/Product.swift

```diff
@@ -9,15 +9,16 @@ public final class Product: CustomStringConvertible {
     public let identifier: String
     public let title: String
     public let info: String
-    public let price: String?
+    public let price: String
+    public let period: SKProductSubscriptionPeriod.Period
```

Comment:
The same is for period. One-time purchases won't have a period, so it makes sense for it to be nil, instead of being `.unknown` - it's not unknown which period it is, there's just none.